tests/Doctrine/Tests/DBAL/Functional/Schema/PostgreSqlSchemaManagerTest.php

```diff
@@ -514,6 +514,17 @@ public function testAlterTableAutoIncrementIntToBigInt(string $from, string $to,
         self::assertTrue($tableFinal->getColumn('id')->getAutoincrement());
     }
 
+    public function testCommentInTable() : void
+    {
+        $table = new Table('table_with_comment');
+        $table->addColumn('id', 'integer');
+        $table->addOption('comment', 'Foo');
```

Comment:
Could you add some basic control characters here (e.g. `'` and `\`) to make sure introspection doesn't need any extra parsing?

Reply:
Control characters added in https://github.com/doctrine/dbal/pull/3499/commits/4d209a5a4d86efd0559f926997a459fd8e24faa4
The build is still successful :)